src/daft-sql/src/modules/aggs.rs

```diff
@@ -170,7 +173,45 @@ Example:
     │ ---   │
     │ Int64 │
     ╞═══════╡
-    │ 1     │
+    │ 2     │
+    ╰───────╯
+    (Showing first 1 of 1 rows)";
+
+    pub(crate) const COUNT_DISTINCT_DOCSTRING: &str =
+        "Counts the number of distinct, non-null elements in the input expression.
+
+Example:
+
+.. code-block:: sql
+    :caption: SQL
+
+    SELECT count(distinct x) FROM tbl
+
+.. code-block:: text
+    :caption: Input
+
+    ╭───────╮
+    │ x     │
+    │ ---   │
+    │ Int64 │
+    ╞═══════╡
+    │ 100   │
+    ├╌╌╌╌╌╌╌┤
+    │ 200   │
+    ├╌╌╌╌╌╌╌┤
+    │ null  │
+    ╰───────╯
+    (Showing first 3 of 3 rows)
+
+.. code-block:: text
+    :caption: Output
+
+    ╭───────╮
+    │ x     │
+    │ ---   │
+    │ Int64 │
+    ╞═══════╡
+    │ 2     │
```

Comment:
The example table which is given here isn't exactly "helpful" when trying to show the whole point of count-distinct; the original table has distinct elements in it already.

I would like to the second element in the table (`200 -> 100`), which would showcase non-uniqueness and how count-distinct will not count duplicates. However, every other example in this uses the same 3 values: `100, 200, NULL`.

Will it be possible for me to deviate from this trend and have a table consisting of `100, 100, NULL`?

Reply:
I think that's fine. Maybe even have four rows (100, 100, 200, Null)

Reply:
Sweet. Updated example.